skyportal/tests/api/test_sources.py

```diff
@@ -346,3 +346,70 @@ def test_source_notifications_unauthorized(
     assert data["message"].startswith(
         "Twilio Communication SMS API authorization error"
     )
+
+
+def test_token_user_source_summary(
+    public_group, public_source, view_only_token_two_groups, public_group2
+):
+
+    now = datetime.utcnow().isoformat()
+
+    status, data = api(
+        "GET",
+        f"sources?saveSummary=true&group_ids={public_group.id}",
+        token=view_only_token_two_groups,
+    )
+    assert status == 200
+    assert "sources" in data['data']
+    sources = data['data']['sources']
+
+    assert len(sources) == 1
+    source = sources[0]
+    assert 'obj' not in source
```

Comment:
Obj wouldn't be present anyways, right? Each dict in `sources` would just have all the obj data (ra, dec, mag, magerr, etc)

Reply:
nice catch, thanks. fixed